Gemfile

```diff
@@ -54,7 +62,10 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'pry', '~> 0.10.4'
+  gem 'better_errors'
 
+  gem 'rb-readline', platforms: :ruby, install_if: os_is(/darwin/)
```

Comment:
cruby 2.3.1 is broken on OS X (for at least some people, myself included) with linker errors to unresolved references in readline.dylib. AFAIK mruby, rubinius, jruby etc isn't.

tl;dr I need it to make ruby work, folks on linux don't. I don't think it makes sense to force it onto anyone else, including possibly anyone who wants to pull this project down and try it on non-cruby on OS X. It makes sense in my mind to clearly document that this is a specific platform/os combination install. Removing platforms: and just using your regex would install it more broadly than it needs to be.

As for the method, same argument. It better semantically documents what's going on here than an inline regex.